content/02.manuscript.md

```diff
@@ -1,12 +1,54 @@
+## Introduction
 
-## Title 1
+- Importance of computation in biological research and scope.
+- Importance to reproducibility in computational biologt.
+- Mentioning previous literature and some of their major takeaways and topics not fully covered in here (what kind of languages to learn, advanced text editors/IDEs, 
+- The computational biology best-practices continuum: reproducibility at the personal level, sharing our research with others and over time.
+- Addressing the audience: computational biologists researchers – data analysts, workflow designers, software developers, mathematical modelers, etc.
+- Structure of this manuscript: mention Figure 1 and Figure 2.
+  - **Figure 1:** Funnel structure for better computational biology.
+  - **Figure 2:** Types of computational biology projects per level.
 
-Some text.
+## Level 1: Individual Research
 
-## Title 2
+Goal: How should you manage your computational biology project?
 
-More great stuff.
+Topics:
+- Literate programming: R Markdown, Jupyter Notebooks
+- Version control: Git / GitHub (commits)
+- Software versioning and environment managers (Conda, python-env)
+- Modularize, snippets
+- Coding style: variable naming, linter, pep8, commenting
+- Programming practices: paradigms (object-oriented, procedural), assertions, pair-programming
 
-### Subtitle
+## Level 2: Collaboration
 
-Details
+Goal: How to allow your collaborators to reproduce and interact with your research/software?
+
+Topics:
+- Sharing notebooks: R Markdowns and Jupyter Notebooks (Binder, Google CoLab)
+- Sharing apps: Shiny apps, Dashboard.
+- GitHub (branching, pull requests)
+- Workflow automation: Snakemake (NextFlow, Make, Bash script)
+- Sharing data and metadata
+
+## Level 3: Community
+
+Goal: How to develop and mantain a computational biology project with community feedback over time?
+
+Topics:
+- GitHub releases and semantic versioning
+- Git Flow, GitHub Issues
+- Continous integration and unit tests
+- Dependencies per user: pip-tools
+- Sharing software as Python packages, Conda/Bioconda or containers (Docker, Singularity)
+- Include license (MIT) and DOIs
+- Documentation: read the docs.
+
+Note: Maybe add software development methodologies (Agile, Scrum, Sprints).
+
+## All three levels in action
+
+- Example of computational biology project 1: RNA-seq analysis (workflow)
+- Example of computational biology project 2: Genome-scale metabolic model (community resource)
```

Comment:
not sure if a GEM intended to be a "community resource" fits this example, as then they should be treated more as computational tools... how about instead:

```suggestion
- Example of computational biology project 2: Genome-scale metabolic model (systems biology project)
```